packages-next/fields-document/ARCHITECTURE.md

```diff
@@ -0,0 +1,53 @@
+# Document editor architecture
+
+The document editor is using [Slate](https://github.com/ianstormtaylor/slate). To learn about Slate generally, go through https://docs.slatejs.org/walkthroughs and https://docs.slatejs.org/concepts. The sections on [Nodes](https://docs.slatejs.org/concepts/02-nodes) and [Locations](https://docs.slatejs.org/concepts/03-locations) contain especially important terminology.
+
```

Comment:
Could you add some brief notes on the data format? I think having a simple example of what a basic document looks like would be really helpful here. Something like...

```suggestion
## Data storage

The document field type stores its data as a JSON blob, representing the Slate formatted data.
A trivial document with a single heading and paragraph would like like

... insert JSON here
```